collections/choosing-projects/index.md

```diff
@@ -6,6 +6,9 @@ items:
  - Homebrew/brew
  - https://www.youtube.com/embed/dSl_qnWO104
  - public-apis/public-apis
+ - firstcontributions/first-contributions
+ - vlang/coreutils
+ - vlang/vinix
```

Comment:
These don't seem to have many issues, and no issues labeled as "good first issue". Any other reason why they should be included in this list as a good first open source projects?
```suggestion
```

Reply:
They are new projects, both striving to be easily understood. Coreutils is made to be easy to claim a command and commit to.